src/test/java/dk/sdu/se_f22/sortingmodule/range/rangefilter/RangeFilterTest.java

```diff
@@ -13,8 +13,8 @@
 
 class RangeFilterTest {
     private RangeFilterInterface rangeFilterInterface;
-    private List<RangeSearchResultMock> mockResults;
-    private List<RangeFilter> mockFilters;
+    private List mockResults;
+    private List mockFilters;
```

Comment:
What's this about?

Reply:
Range has their own product class, and can therefore not interact with the product used by us.

Reply:
It seems to me very weird that you have to the RangeFilter from the type in the list since the whole purpose of that test is to test the functionality of the RangeFilter class. Besides this test has been reworked to use ProductHit objects instead of RangeSearchResultMock. 
Does the test break the compilation if you do not change the types?

Reply:
The whole project will not compile, if i didn't remove the type.
This is due to us now using the Product class - We will today, at the checkpoint, ask which class will be used

Reply:
Could we hold on with merging this till we have spoken tomorrow about the state of Product

Reply:
Will do